### ISSUE TITLE
Configurable fast sampling rate

### ISSUE THREAD

**Issue by nicolas-f** [Ifsttar/NoiseCapture#238]
Currently the display is refreshed all 125 ms. Some users want to have a real time level with lower display frequency.

Do we let the user select the window time of real time measurement ?

**Comment by Picaut** [comment]
125ms is quite fast. I'm not sure that lower display frequency is a solution. The problem may be due to the smartphone.

**Comment by nicolas-f** [comment]
Yes its too fast for the display. Users want to have the time to read the value, so may be an option to switch to slow (1s) display while measuring (displaying the recorded values).

**Comment by nicolas-f** [comment]
We could change the "window" option for something more user friendly like:

"Real time display mode"
- Fast 125ms Rectangular
- Fast 125ms Hann
- Slow 1s Hann

**Comment by Picaut** [comment]
OK, but not a priority

**Comment by nicolas-f** [comment]
We could use IEC 61672-1 specification in order to fit to the displaying of noise level as in standard noise level meter:

https://www.nti-audio.com/en/support/know-how/fast-slow-impulse-time-weighting-what-do-they-mean


 _When the sound is suddenly switched OFF, the displayed 'F' level decays at a rate of 34.7 dB/s, while the displayed 'S' level decays at a rate of 4.3 dB/s._

**Comment by nicolas-f** [comment]
It must be done with something related to a weighted average but I don't have any clue to the coefficients for the implementation. @pierromond

**Comment by nicolas-f** [comment]
Implemented in
https://github.com/nicolas-f/NoiseCaptureKotlin/commit/fbf3d36592ee5e33aaca10db0e86567cde1dcc54
